src/styles/main.scss

```diff
@@ -1,3 +1,226 @@
+/* #region Normalize */
 body {
+  font-family: "Roboto", Helvetica, sans-serif;
```

Comment:
have multiple blocks , use BEM method
for single product card you had 2 blocks
here you can have 
header, nav, card, start ....